.env.example

```diff
@@ -85,5 +85,8 @@ ENABLE_ITEM_EMBED_CODE=1
 # Enable recommendations
 # ENABLE_RECOMMENDATIONS=0
 
+# Enable announcer on route changes
+ENABLE_ANNOUNCER=1
```

Comment:
Rename this to `ENABLE_VUE_ANNOUNCER`.

Also, while this is a work-in-progress, in the example file it should disabled by default, i.e.:
```
# ENABLE_VUE_ANNOUNCER=0
```